metadata-ingestion/docs/sources/mssql/mssql_recipe.yml

```diff
@@ -9,6 +9,13 @@ source:
     username: user
     password: pass
 
+    # Options
+    # See https://python-tds.readthedocs.io/en/latest/pytds.html#pytds.connect
+    options:
+      connect_args:
+        cafile: server-ca.pem
+        validate_host: true
```

Comment:
Can we keep this section commented out in the example recipe, and add a comment saying "uncomment if you need to use encryption with pytds"?